src/apps/active-preview/components/Frame/Frame.js

```diff
@@ -12,21 +12,41 @@ import pixel7cam from "../../../../../public/images/pixel-7-camera.png";
 import styles from "./Frame.less";
 export function Frame(props) {
   const [frameLoading, setFrameLoading] = useState(true);
+  const [routePath, setRoutePath] = useState("");
 
   useEffect(() => {
     setFrameLoading(true);
-  }, [props.device]);
+  }, [props.route]);
```

Comment:
Have you verified if this doesn't affect the functionality when changing device previews?